internal/sqlite/writer/writer.go

```diff
@@ -235,8 +235,8 @@ func resolvePlaces(
 		if _, ok := placeToDCID[entity.GetNode()]; ok {
 			continue
 		}
-		// TODO(ws): Handle the case with multiple candidates.
 		if len(entity.GetCandidates()) > 0 {
+			// The resolve API sorts candidates already, so we pick the first one.
```

Comment:
Very soon we want to read the type of place from user config. Does each item here have the corresponding type?

Reply:
Yes, each item has a 'dominantType' field, so we can actually do filtering here if needed in the future.